src/styles/utils/variables.scss

```
@@ -0,0 +1,7 @@
+$heightCard: 300px;
+$widthCard: 200px;
+$media:
+  "1px" 1,
+  "488px" 2,
+  "768px" 3,
+  "1024px"  4;
```

No need to specify (min-width: 1px) in $media, you've already created grid with 1 column and in media query you should change amount of columns.
```suggestion
$media:
  "488px" 2,
  "768px" 3,
  "1024px"  4;
```

---

fixed
Ty for rewiew:)